### DISCUSSION TITLE
what does geodata-mode do?

### DISCUSSION THREAD

**Discussion by Azadzadeh** [MetaCubeX/mihomo#795] [Q&A]
there is no explanation for `geodata-mode: true` in the docs (https://wiki.metacubex.one/config/)
what does this option do?

**Comment by vernesong** [comment]
https://wiki.metacubex.one/config/rules/geox/

**Reply by vernesong** [reply]
https://wiki.metacubex.one/config/rules/geox/#geosite

**Reply by Azadzadeh** [reply]
@vernesong  yeah, but what does `geodata-mode: true` do specifically? even when it's not set, we can still use GEOSITE and GEOIP in rules.

**Comment by aoxiangtianji** [comment]
[GEO数据模式](https://wiki.metacubex.one/config/general/#geo)[¶](https://wiki.metacubex.one/config/general/#geo)

更改geoip使用文件,mmdb或者dat,可选 true / false,true为dat,此项有默认值 false

Change the file used by geoip, mmdb or dat, optional true / false, true is dat, this item has a default value of false
